### PR TITLE
makes the map for storing the color offset for each depth pixel a function parameter in order to make the user decide the allocation policy

### PR DESCRIPTION
**Pull request by giacomodabisias**
makes the map for storing the color offset for each depth pixel a function parameter in order to make the user decide the allocation policy

### CODE REVIEW

File: src/registration.cpp [hunk 1]
```diff
@@ -100,9 +100,10 @@ void Registration::apply( int dx, int dy, float dz, float& cx, float &cy) const
  * @param [out] registered Image color image for the depth data (512x424).
  * @param enable_filter Use a depth buffer to remove pixels which are not visible to both cameras.
  * @param [out] bigdepth If not \c NULL, mapping of depth onto colors (1920x1082 'float' frame).
+ * @param [out] bigdepth If not \c NULL, map for storing the color offset for each depth pixel.
```
Comment (xlz) [inline src/registration.cpp]: bigdepth -> color_depth_map. And its required size?

**Comment by floe** [comment]
Since this doesn't really change the interface or the internal operation and just provides an option which some people may find useful, I'll include this for 0.2.

**Comment by giacomodabisias** [comment]
Thanks

**Comment by xlz** [comment]
This breaks the ABI. If this is to be merged, it is before 0.1.

**Comment by floe** [comment]
OK, not a problem. Fine for 0.1, too.

**Comment by ludiquechile** [comment]
Hi I think there's an issue on this commit, it appears that the method apply() of class RegistrationImpl is missing the parameter color_depth_map.

**Pull request merged**